paypack/event.go

```diff
@@ -0,0 +1,40 @@
+package paypack
+
+import (
+	"context"
+	"time"
+)
+
+type Event struct {
+	ID        string
+	Data      EventData
```

Comment:
Make this a `Data []byte` cause the data may not always be a transaction payload. and the unmarshal it into the `Transaction` type instead of using yet another type(`EventData`) that duplicates `Transaction`.

Reply:
`EventData` provide extract information like transaction `status`, `processed_at` etc... but `Transaction` give us detailed information that why I used `EventData` type.

Reply:
You can add those to `Transaction` as pointers and add `omitempty` json tag